### ISSUE TITLE
Unable to change host tcp port on container 

### ISSUE THREAD

**Issue by Grim73** [tborychowski/courier#13]
##Steps to reproduce
---
- copy [docker-compose](https://github.com/tborychowski/courier/tree/master#run)
    - https://github.com/tborychowski/courier/blob/39dc947bde7602df6f7d3fcaddf3ae7450d3a0cd/README.md?plain=1#L26-L38
- changed port to `6000:6000` on L34:
    - https://github.com/tborychowski/courier/blob/39dc947bde7602df6f7d3fcaddf3ae7450d3a0cd/README.md?plain=1#L34
- Run `docker compose up -d` 
- check `docker ps -a`
    - `4da0c78880aa   tborychowski/courier         13 seconds ago   Up 12 seconds             3000/tcp, 0.0.0.0:6000->6000/tcp, :::6000->6000/tcp` 

## Investigation
---
 I noticed in the [server.js](https://github.com/tborychowski/courier/blob/master/server/index.js) on L9 has its port set to `3000`.

https://github.com/tborychowski/courier/blob/39dc947bde7602df6f7d3fcaddf3ae7450d3a0cd/server/index.js#L9

I am unsure how to correct this, as I am new to docker, but any help or guidance would be greatly appreciated

**Comment by Grim73** [comment]
## My `docker-compose.yaml`


```
version: '3.7'
services:
  courier:
    container_name: courier
    image: tborychowski/courier
    restart: unless-stopped
    ports:
      - "6000:6000"
    environment:
      - NODE_ENV=production
      - TOKEN=<Tokenid>
```

**Comment by tborychowski** [comment]
Hi,
Do you want to change the internal port or external one?
E.g.
- my app runs on 3000 by default
- if you wish to change that (no idea why would you care) - is to provide the env variable `PORT` in the `environment` section, e.g.:

```yaml
---
services:
  courier:
    container_name: courier
    image: tborychowski/courier
    restart: unless-stopped
    environment:
      - PORT=6000
      - NODE_ENV=production
      - TOKEN=<Tokenid>
    ports:
      - "6000:6000"
```
and then your ports will work.

However, usually you don't change the internal port of the app in the container (as you don't need to care about it), you just map the internal port to external (one which you will actually use), like so:
```yaml
   ports:
     - external:internal
```

So - in your example, if you want the app to be available in your browser at http://serverIP:6000, you do this:


```yaml
---
services:
  courier:
    container_name: courier
    image: tborychowski/courier
    restart: unless-stopped
    environment:
      - NODE_ENV=production
      - TOKEN=<Tokenid>
    ports:
      - "6000:3000"
```

**Comment by Grim73** [comment]
Honestly, I feel like an absolute fool.. I got this working using your instruction.  I appreciate the time you took to ELI5. closing the issue, and will keep this in mind in the future.
